StripeConnect/StripeConnect/Source/Internal/Webview/MessageSenders/UpdateConnectInstanceSender.swift

```diff
@@ -9,9 +9,9 @@ import Foundation
 
 /// Updates appearance and locale options.
 struct UpdateConnectInstanceSender: MessageSender {
-    struct Payload: Codable, Equatable {
+    struct Payload: Encodable, Equatable {
         let locale: String
-        // TODO: Add appearance here.
+        private(set) var appearance: Appearance = .default
```

Comment:
Instead of defaulting appearance to `.default`, we could default it to `nil` since appearance and locale are both optional fields in the JS object passed to web.

I think this is better because it will prevent us from unintentionally resetting the appearance to the default when we actually intend to not set it.
```suggestion
        private(set) var locale: String?
        private(set) var appearance: Appearance?
```

Reply:
Part of the public api for `EmbeddedComponentManager` is making appearance not optional and defaulted to .default. With that in mind is there ever a time where we will send a message that doesn't have an appearance? It seems like the right thing to do here is to not make make appearance optional but also not provide a default value that way it is required that we send some value (ideally always from EmbeddedComponentManager).
```
    public init(apiClient: STPAPIClient = STPAPIClient.shared,
                appearance: Appearance = .default,
                fonts: [CustomFontSource] = [],
                fetchClientSecret: @escaping () async -> String?)
```

Reply:
Correct, I think the public SDK should always default to the `default` value.

We could theoretically set just `locale` and not `appearance` when the locale is updated (or visa versa), but it's also valid for us to set both any time either is updated since we'll have both of them cached on the view.

My comment was intended to be specific to setting the default value in `UpdateConnectInstanceSender`, and not `EmbeddedComponentManager` with the rationale that we could unintentionally only set `locale` when instantiating `UpdateConnectInstanceSender` and forget to explicitly set appearance to the view's current appearance value. This would result in the appearance getting reset to the default value whenever the app's locale is updated.

Making both `appearance` and `locale` required init arguments in `UpdateConnectInstanceSender` also addresses the issue because it compiler-enforces us to always pass explicit values.